qrenderdoc/Windows/PixelHistoryView.cpp

```diff
@@ -746,7 +748,7 @@ void PixelHistoryView::startDebug(EventTag tag)
 
   m_Ctx.Replay().AsyncInvoke([this, &trace, &done, tag](IReplayController *r) {
     trace = r->DebugPixel((uint32_t)m_Pixel.x(), (uint32_t)m_Pixel.y(),
-                          m_Display.subresource.sample, tag.primitive);
+                          m_Display.subresource.sample, tag.primitive, m_Display.subresource.slice);
```

Comment:
Render target array slices and multiview view indices are unfortunately not interchangeable like this.

Multiview will broadcast to corresponding indices within the attached image view, but the image view may not have its first array slice at slice 0. That will be a common case, but not the only case. If you have an image view bound which has slice 3 as the first slice and 4-way multiview then view index 0 is slice 3, view index 1 is slice 4, etc.